specification/search/data-plane/autorest-custom-directives.md

```diff
@@ -0,0 +1,20 @@
+```yaml
```

Comment:
We may need to get rid of this file, subject to the conclusion of [https://github.com/Azure/azure-rest-api-specs/pull/7923#discussion_r355606332](https://github.com/Azure/azure-rest-api-specs/pull/7923#discussion_r355606332)

Reply:
@sakintoye change-object-ref-to-string is still being used